.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '10'
+          node-version: '16'
```

Comment:
This looks like continuation of #56 ...so I'd call the fact that we were able to bump the node version there and miss the CI configuration tech debt. Let's not hold back this PR but let's figure out how to ensure when we bump node versions in the future we get everything in one go–perhaps we should be looking at how other projects use a `.tool-versions` file as a source of truth that gets consumed by other systems, like docker, ci, npm, etc.